src/features/prompts/components/prompt-detail.tsx

```diff
@@ -112,6 +115,10 @@ export const PromptDetail = (props: PromptDetailProps) => {
               )}
             </div>
           </div>
+
+          {prompt.config && Object.keys(prompt.config).length > 0 && (
```

Comment:
Object.keys does not work if JSON is not of type Object. JSON could also be an array or a scalar (number/string/…). I’d suggest to change to prompt.config !== {}

Reply:
Isn't the problem with `prompt.config !== {}` that in TS we don't compare the properties but the reference in memory and therefore the evaluation would always be true?

To better understand the background of the concern: The case that the config is array or scalar would be only possible for API ingested prompts right? Prompts edited via UI are always an object.

My first thought to solve this would actually be to write a custom function like:
`function isJsonEmpty(json: jsonSchema): boolean {
  if (json === null || json === undefined) {
    return true;
  }
  if (Array.isArray(json)) {
    return json.length === 0;
  }
  if (typeof json === 'object') {
    return Object.keys(json).length === 0;
  }
  // For scalar values
  return false;
}`

@marcklingen wdyt?

Reply:
For simplicity we can also always show the config and remove this check.

Reply:
After some further research I realized that:
Object.keys().length also works for arrays and strings.
Therefore the only cases currently not covered would be int, floats, booleans as null and undefined are prohibited by the schema. Should we support those edge cases?